src/index.tsx

```diff
@@ -13,50 +27,67 @@ type ProvidedContext = {
   loading: boolean;
   prefab: typeof prefab;
   keys: string[];
+  settings: SharedSettings;
 };
 
-const defaultContext: ProvidedContext = {
+export const defaultContext: ProvidedContext = {
```

Comment:
I'm not seeing where we need this exported anymore

Reply:
👍